models/device.go

```diff
@@ -29,5 +29,12 @@ type ProtocolProperties map[string]any
 // AdminState controls the range of values which constitute valid administrative states for a device
 type AdminState string
 
+func AssignAdminState(dtoAdminState string) AdminState {
```

Comment:
add go docs for the public function
// AssignAdminState provides a default value "UNLOCKED" to AdminState